package.json

```diff
@@ -153,6 +153,7 @@
   },
   "vjsstandard": {
     "ignore": [
+      "build",
```

Comment:
What type of errors were you seeing? I don't have any issues running the linter locally.

Reply:
My local `build/` doirectory was somehow outdated. This is not needed so I removed it.